kubeyaml should not be a prerequisite for running the tests

currently when running tests, there are errors unless you have [`kubeyaml`](https://github.com/squaremo/kubeyaml) in your path:

```
$ go test github.com/weaveworks/flux/daemon
--- FAIL: TestDaemon_Release (0.21s)
        daemon_test.go:838: applying changes: exec: "kubeyaml": executable file not found in $PATH
--- FAIL: TestDaemon_PolicyUpdate (0.22s)
        daemon_test.go:838: exec: "kubeyaml": executable file not found in $PATH
--- FAIL: TestDaemon_SyncStatus (0.26s)
        daemon_test.go:838: applying changes: exec: "kubeyaml": executable file not found in $PATH
--- FAIL: TestDaemon_JobStatusWithNoCache (0.13s)
        daemon_test.go:838: exec: "kubeyaml": executable file not found in $PATH
--- FAIL: TestDaemon_Automated (10.17s)
        daemon_test.go:821: Waiting for image tag: "2"
--- FAIL: TestDaemon_Automated_semver (10.11s)
        daemon_test.go:821: Waiting for image tag: "3"
event="Sync: 9d04903, default:daemonset/init, default:deployment/helloworld, default:deployment/list-deploy, default:deployment/locked-service, default:deployment/multi-deploy, default:deployment/semver, default:deployment/test-service, default:service/list-service, default:service/multi-service" logupstream=true
event="Sync: cba0b77, default:deployment/helloworld" logupstream=true
FAIL
FAIL    github.com/weaveworks/flux/daemon       23.027s
```

It seems reasonable enough to me that this can/should be automated out.  Seems like we're close to that already anyway.  There's a `/bin` folder in the Flux repo that contains a dockerfile for running `kubeyaml`:
```dockerfile
#!/bin/sh
docker run --rm -i quay.io/squaremo/kubeyaml:0.5.1 "$@"
```

The code in question is in `kubeyaml.go`:

```
func execKubeyaml(in []byte, args []string) ([]byte, error) {
	cmd := exec.Command("kubeyaml", args...)
	...
	err := cmd.Run()
}
```

I don't have a solution off hand, but it seems reasonable that this is something you shouldn't need to install in your PATH before running tests.  I could happily be persuaded otherwise.  If this is intended, then I will update the documentation to list `kubeyaml` being in your path a requirement for the path.

---

For reference, here is what the `Dockerfile.flux` does:
```dockerfile
# Get the kubeyaml binary (files) and put them on the path
COPY --from=quay.io/squaremo/kubeyaml:0.5.1 /usr/lib/kubeyaml /usr/lib/kubeyaml/
ENV PATH=/bin:/usr/bin:/usr/local/bin:/usr/lib/kubeyaml
```

--- a/CONTRIBUTING.md
+++ b/CONTRIBUTING.md
@@ -58,7 +58,7 @@ This is a rough outline of how to prepare a contribution:
 Refer to the [building doc](site/building.md) to find out how to build from
 source.
 
-Refer to the [up-and-running with Flux Development](site/get-started-developing.md) doc for a walkthrough on developing Flux locally.
+Refer to the [Get Started Developing](site/get-started-developing.md) guide for a walkthrough on developing Flux locally.
 
 ### How to run the test suite
 

--- a/site/get-started-developing.md
+++ b/site/get-started-developing.md
@@ -1,6 +1,6 @@
 # Get Started Developing Flux
 
-This tutorial is going to walk you through one possible workflow for making a small (actually, tiny) change to Flux and building and testing that change locally.
+This guide shows a workflow for making a small (actually, tiny) change to Flux, building and testing that change locally.
 
 > ### TL;DR
 >
@@ -38,6 +38,11 @@ Now that we know everything is working with `flux-getting-started`, we're going 
     - [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
     - [Docker](https://docs.docker.com/install/)
     - [Go](https://golang.org/doc/install)
+
+1. Configure your enviroment so you can run tests.  Run:
+    ```sh
+    make test
+    ```
 
 1. We want to make sure we're starting fresh.  Tell Minikube to clear any previously running clusters.
     ```sh

